### PR TITLE
docs: Add swagger docs

### LINKED ISSUE #100: Document IntelliTect.Coalesce.Swashbuckle
As the title states. Pretty simple to use it, but still needs to be written down somewhere beyond just "look at Coalesce.Web's Startup.cs"

### PR DESCRIPTION
**Pull request by meghanmae**
fixes #100

### CODE REVIEW

File: docs/topics/coalesce-swashbuckle-without.png
Comment (ascott18) [inline docs/topics/coalesce-swashbuckle-without.png:1]: Both images are showing as invalid/corrupted files on Github

File: docs/topics/coalesce-swashbuckle.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+# Coalesce Swashbuckle
+
+When using Coalesce to generate API endpoints the default Swagger generation can sometimes result in verbose and confusing API definitions, especially when dealing with DataSources and Behaviors. To address these issues, the `IntelliTect.Coalesce.Swashbuckle` package offers enhancements for Swagger definitions, making your Coalesce-generated APIs clearer and more manageable.
```
Comment (ascott18) [inline docs/topics/coalesce-swashbuckle.md:3]:
```suggestion
When using Coalesce to generate API endpoints, the default Swagger generation can sometimes result in verbose and confusing API definitions, especially when dealing with DataSources and Behaviors. To address these issues, the `IntelliTect.Coalesce.Swashbuckle` package offers enhancements for Swagger definitions, making your Coalesce-generated APIs clearer and more manageable.
```

File: docs/topics/coalesce-swashbuckle.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+**Purpose:** Helps refine the operational details in the Swagger documentation for Coalesce-generated endpoints.
+
+**Functionality:** The filter processes various aspects of the API operations to provide cleaner generated API docs:
+
+- DTO Parameters: Adjusts the representation of DTO parameters to ensure only relevant properties are included.
+
+- Data Sources: Enhances the description and the schema of data source parameters so they are accurately represented.
+
+- Standard Parameters: It removes unnecessary parameters related to behaviors and data sources, improving the clarity of the generated swagger API documentation.
```
Comment (ascott18) [inline docs/topics/coalesce-swashbuckle.md:67]: I think just a simple high level summary of the effects is OK. We don't need to detail the exact functionality of each individual Filter implementation class - we can consider those as internal implementation details. The user will get a good enough idea of the effects.

```suggestion
The primary effect is an adjustment of parameter definitions to account for Coalesce's custom model binders that create instances of Data Sources and Behaviors on each request. These parameters will be updated in the OpenAPI document to account for data source parameters, filter parameters, and other model-specific customizations.
```

File: docs/topics/coalesce-swashbuckle.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
+# Coalesce Swashbuckle
```
Comment (ascott18) [inline docs/topics/coalesce-swashbuckle.md:1]:
```suggestion
# OpenAPI/Swagger
```

File: docs/topics/coalesce-swashbuckle.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
```
Comment (ascott18) [inline docs/topics/coalesce-swashbuckle.md:2]: Lets update all references to "Swagger" to "OpenAPI" where appropriate, or to "Swashbuckle" when referring to the specific nuget packages.

Swagger is a suite of tools and services created and sold by SmartBear around the OpenAPI standard. We should include maybe a short blurb in the first paragraph for searching purposes, but nothing else aside from the references in Swashbuckle's method names (which are named so for legacy/compatibility reasons). 

Something like "... OpenAPI (sometimes referred to as its pre-2015 name "Swagger") ..."

File: docs/topics/coalesce-swashbuckle.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+## Visual Comparison 
+
+To illustrate the impact of the `IntelliTect.Coalesce.Swashbuckle` package, let's examine the Patient model and its representation in Swagger.
+
+```c#
+public class Patient
+{
+    public int PatientId { get; init; }
+    public DateTime NextAppoitment { get; set; }
```
Comment (ascott18) [inline docs/topics/coalesce-swashbuckle.md:77]:
```suggestion
    public DateTime NextAppointment { get; set; }
```

File: docs/topics/coalesce-swashbuckle.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
...
+    // Additional parameters
```
Comment (ascott18) [inline docs/topics/coalesce-swashbuckle.md:78]:
```suggestion
    // Additional properties
```

**Review by ascott18 — CHANGES_REQUESTED** [review]

File: docs/topics/coalesce-swashbuckle-without.png
Comment (meghanmae) [reply docs/topics/coalesce-swashbuckle-without.png:1]: Made them into jpgs rather than pngs, they seem to be happier!

**Pull request merged**